client/jetpack-cloud/sections/partner-portal/credit-card-fields/index.tsx

```diff
@@ -97,8 +103,8 @@ export default function CreditCardFields() {
 				/>
 
 				<SetAsPrimaryPaymentMethod
-					isChecked={ useAsPrimaryPaymentMethod }
-					isDisabled={ isDisabled }
+					isChecked={ useAsPrimaryPaymentMethod || storedCards.length === 0 }
```

Comment:
Is it guaranteed that `useAsPrimaryPaymentMethod` will be true by default? If it's false, the storedCards check will make the checkbox seem checked but from a store perspective, the value will remain false.